File: spec/factories/veiculos.rb
FactoryGirl.define do
  factory :veiculo do
    disponivel_viagem true
    modelo "MyString"
    ano 2017
    chassi "MyString"
    placa "MyString"
    renavam "MyString"
    motor "MyString"
    data_compra "2017-02-02"
    nf "MyString"
    ocupantes 1
    media 1
    combustivel 'gasolina'
  end
end

File: app/assets/javascripts/veiculos.js
$(document).on('ready', function(){
  var campos = ["modelo","ano","placa","combustivel","ocupantes","media","disponivel_viagem"];
  carrega_datatable_filtro("veiculos","veiculo", campos, []);

  $(document).off("dblclick", "#veiculos-table tr[id^=veiculo]")
  $(document).on("dblclick", "#veiculos-table tr[id^=veiculo]", function(){
    var id = $(this).attr("id");
    var url = "/veiculos/" + id.replace("veiculo_", "") + "/editar"
    $.ajax({ type: "GET", url: url });
  });

});
